docs/datasets/lithuanian.md

```diff
@@ -0,0 +1,409 @@
+# 🇱🇹 Lithuanian
+
+This is an overview of all the datasets used in the Lithuanian part of EuroEval. The
+datasets are grouped by their task - see the [task overview](/tasks) for more
+information about what these constitute.
+
+
+## Sentiment Classification
+
+### Lithuanian Emotions
+
+This dataset is a combination of machine translated versions of the [GoEmotions dataset](https://arxiv.org/abs/2005.00547) and the [Kaggle emotions dataset](https://www.kaggle.com/datasets/nelgiriyewithana/emotions). GoEmotions consists of English Reddit comments and the Kaggle dataset contains English Twitter messages. Both datasets have been machine translated to Lithuanian.
```

Comment:
Preference for using DOI links when available, as they're less prone to link rot.

```suggestion
This dataset is a combination of machine translated versions of the [GoEmotions dataset](https://doi.org/10.48550/arXiv.2005.00547) and the [Kaggle emotions dataset](https://www.kaggle.com/datasets/nelgiriyewithana/emotions). GoEmotions consists of English Reddit comments and the Kaggle dataset contains English Twitter messages. Both datasets have been machine translated to Lithuanian.
```